src/processing/mangekanter/mangekanter.md

```diff
@@ -0,0 +1,468 @@
+---
+title: Kanter, kanter, mange mangekanter
+level: 2

```

Comment:
Jeg har bare gjort kurs med arrayer og/eller polygoner på videregående elever og The Gathering. Det er jo et litt annet publikum, men arrayer og løkker pleier å gå fint på et tre-timers introkurs (helt fra bunnen av). Noen ganger bruker jeg også lister av typen ArrayList, men da har jeg som regel introdusert OOP først.

Trigonometri, regulære mangekanter og stjerner, kan sikkert gjøres til en egen oppgave.